core/src/test/scala/com/avast/cloud/metrics/datadog/impl/TimerImplTest.scala

```diff
@@ -0,0 +1,44 @@
+package com.avast.cloud.metrics.datadog.impl
+
+import java.util.concurrent.TimeUnit
+
+import cats.effect.{ Clock, IO }
+import com.avast.cloud.metrics.datadog.api.Tag
+import com.timgroup.statsd.StatsDClient
+import org.mockito.scalatest.MockitoSugar
+import org.scalatest.{ Assertions, BeforeAndAfter, FlatSpec }
+
+class TimerImplTest extends FlatSpec with MockitoSugar with BeforeAndAfter with Assertions {
```

Comment:
why not `fixture.FlatSpec`?

Reply:
this approach is more convenient for multiple fixtures